LDAR_Sim/model_code/methods/deployment/stationary_crew.py

```diff
@@ -0,0 +1,58 @@
+from datetime import timedelta
```

Comment:
same as above. Consider name change. I wonder if we should put everything related to scheduling in the same folder or same file called scheduling

Reply:
same as above